src/implementations/twig/components/link/link.html.twig

```diff
@@ -171,7 +182,7 @@
         {%- include '@ecl/icon/icon.html.twig' with _icon|merge({
           icon: icon|default({}),
           extra_classes: _icon_extra_classes,
-          extra_accessibility: extra_accessibility|default({}),
+          extra_accessibility: _extra_accessibility|default({}),
```

Comment:
we don't need anymore the default here, since we already do that at the top of the template